titus-ext/aws/src/main/java/com/netflix/titus/ext/aws/appscale/AWSAppScalingConfig.java

```diff
@@ -19,6 +19,9 @@
 import com.netflix.archaius.api.annotations.PropertyName;
 
 public interface AWSAppScalingConfig {
+    String SERVICE_NAMESPACE = "custom-resource";
+    // AWS requires this field be set to this specific value for all application-autoscaling calls.
+    String SCALABLE_DIMENSION = "custom-resource:ResourceType:Property";
```

Comment:
If these are constants then I think you should just move them to the client and make them `public static final`.